Code/Engine/RendererFoundation/RendererFoundationDLL.h

```diff
@@ -281,7 +281,8 @@ struct ezGALCompareFunc
 
   enum Enum
   {
-    Never = 0,
+    None = 0,
+    Never,
```

Comment:
What is the difference between none and never?

Reply:
We currently don't have an "off" state for this. D3D doesn't care, but other APIs do. So this is for the situation where the comparison isn't used, at all. It could also be stored separately, but that feels unnecessary.

Reply:
Always is your "off" state

Reply:
WebGPU differentiates between "Always" and "Undefined". In some situations "Always" does make sense (I think), so you have to set that, but if you set something other than "Undefined", it will complain about that, if you use it in situations where you can't compare things.

Reply:
I've reverted this change for now, because it breaks the renderer tests (though not on my machine).